src/figuresClasses.ts

```diff
@@ -1,19 +1,113 @@
+type Shapes = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  shape: Shapes,
```

Comment:
you can create enum for all the shapes)

Reply:
When I wrote code first time I had errors because of it, thats why wrote the long way. But now I refactored code, and everything's ok!